daemon/algod/api/algod.oas2.json

```diff
@@ -23,6 +23,88 @@
   "host": "localhost",
   "basePath": "/",
   "paths": {
+    "/health": {
+      "get": {
+        "tags": [
+          "common"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "schemes": [
+          "http"
+        ],
+        "summary": "Returns OK if healthy.",
+        "operationId": "HealthCheck",
+        "responses": {
+          "200": {
+            "description": "OK."
+          },
+          "401": {
+            "description": "Invalid API Token"
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "type": "string"
+            }
```

Comment:
Not sure in what case it can provide 404. maybe 500? if something really weird is going on.

Reply:
fixed